index.html

```diff
@@ -126,27 +134,27 @@ <h1 class="item1 h1">About Myself</h1>
           <button>Get My Resume</button>
           <div class="abfr">
             <p class="language">Language</p>
-            <img class="abfr1" src="./images/download.png" alt="">
+            <img class="abfr1" src="./images/download (1).png" alt="droupdown icon">
           </div>
           <div class="abla abla1">
-            <img class="abimg" src="./images/Ellipse 1.png" alt="">
+            <img class="abimg" src="./images/Ellipse 1.png" alt="JS image">
             <p>JavaScript</p>
           </div>
           <div class="abla abla2">
-            <img class="abimg" src="./images/Ellipse 1 (1).png" alt="">
+            <img class="abimg" src="./images/Ellipse 1 (1).png" alt="Html image">
             <p>HTML</p>
           </div>
           <div class="abla abla3">
-            <img class="abimg" src="./images/Ellipse 1 (4).png" alt="">
+            <img class="abimg" src="./images/Ellipse 1 (4).png" alt="css image">
             <P>CSS</P>
           </div>
           <div class="abfr">
             <p>FrameWork</p>
-            <img class="abfr1" src="./images/Disabled.png" alt="">
+            <img class="abfr1" src="./images/download.png" alt="arrow icon">
           </div>
           <div class="abfr">
             <p>Skill</p>
-            <img class="abfr1" src="./images/Disabled.png" alt="">
+            <img class="abfr1" src="./images/download.png" alt="arrow icon">
```

Comment:
- [ ] Great job implmenting the changes from previous reviewer. However, there is one small thing you need to work on. The direction of the arrow on the language and the framework and skill are not align with what we have on the Figma design. Kindly look at them and fix.

### Your Design
![your-lang](https://user-images.githubusercontent.com/26597432/227711260-e60d461a-9cc6-4d9f-bc27-d70442cb56e8.png)

### Figma Design
![temp-lang](https://user-images.githubusercontent.com/26597432/227711248-fe201888-4152-4b04-8d86-8f3594ff64f9.png)